### PR TITLE
Dropdown

### PR DESCRIPTION
**Pull request by gazzdingo**
### Features and Changes

- Closes **(add link to issue here)**

### CODE REVIEW

File: packages/front-end/components/Radix/Dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import { DropdownMenu as RadixDropdownMenu } from "@radix-ui/themes";
+// import colors from radix-ui themes
+import type { MarginProps } from "@radix-ui/themes/dist/cjs/props/margin.props";
+type AllowedChildren = React.ReactNode;
+
+type Props = {
+  trigger: React.ReactNode;
+  disabled?: boolean;
+  errorLevel?: "error" | "warning";
+  description?: string;
```
Comment (msamper) [inline packages/front-end/components/Radix/Dropdown.tsx:11]: Are these radix dropdown props? I wasn't able to find them in the API reference here: https://www.radix-ui.com/themes/docs/components/dropdown-menu#root

File: packages/front-end/components/Radix/Dropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  children: AllowedChildren;
+} & MarginProps;
+export function Dropdown({ trigger, children, ...containerProps }: Props) {
+  return (
+    <RadixDropdownMenu.Root {...containerProps}>
+      <RadixDropdownMenu.Trigger>{trigger}</RadixDropdownMenu.Trigger>
+      <RadixDropdownMenu.Content>{children}</RadixDropdownMenu.Content>
```
Comment (msamper) [inline packages/front-end/components/Radix/Dropdown.tsx:18]: Looking at the design, I think we always want the `variant` prop for content to be `"soft"`

File: packages/front-end/components/Radix/Dropdown.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+// import colors from radix-ui themes
+import type { MarginProps } from "@radix-ui/themes/dist/cjs/props/margin.props";
+type AllowedChildren = React.ReactNode;
+
+type Props = {
+  trigger: React.ReactNode;
+  disabled?: boolean;
+  errorLevel?: "error" | "warning";
+  description?: string;
+  children: AllowedChildren;
```
Comment (msamper) [inline packages/front-end/components/Radix/Dropdown.tsx:12]: we should have a color prop here for changing the color on hover for `DropdownMenu.Content`. Looking at the design I think we only want to allow red and violet

**Review by msamper — CHANGES_REQUESTED** [review]

File: packages/front-end/components/Radix/Dropdown.tsx (hunk 3 above)
Comment (gazzdingo) [reply packages/front-end/components/Radix/Dropdown.tsx:15]: color is already on the items them self

File: packages/front-end/components/Radix/Button.tsx [hunk 4]
```diff
@@ -9,6 +9,7 @@ export type Size = "xs" | "sm" | "md" | "lg";
 
 export type Props = {
   onClick?: (() => Promise<void>) | (() => void);
+  preventDefault?: boolean;
```
Comment (gazzdingo) [inline packages/front-end/components/Radix/Button.tsx:12]:
```suggestion
```

File: packages/front-end/components/Radix/Dropdown.tsx [hunk 5]
```diff
@@ -0,0 +1,106 @@
+// radix dropdown wrapper
+import {
+  DropdownMenu as RadixDropdownMenu,
+  Button,
+  Text,
+} from "@radix-ui/themes";
+// import colors from radix-ui themes
```
Comment (gazzdingo) [inline packages/front-end/components/Radix/Dropdown.tsx:7]:
```suggestion

```

File: packages/front-end/components/Radix/Dropdown.tsx [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+}: DropdownContentProps) {
+  return (
+    <RadixDropdownMenu.Sub>
+      <RadixDropdownMenu.SubTrigger>{trigger}</RadixDropdownMenu.SubTrigger>
+      <RadixDropdownMenu.SubContent {...props}>
+        {children}
+      </RadixDropdownMenu.SubContent>
+    </RadixDropdownMenu.Sub>
+  );
+}
```
Comment (gazzdingo) [inline packages/front-end/components/Radix/Dropdown.tsx:57]:
```suggestion
}

```

File: packages/front-end/components/Radix/Dropdown.tsx [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+      disabled={disabled}
+      onSelect={onClick}
+      color={color}
+      shortcut={shortcut}
+      {...props}
+    >
+      {children}
+    </RadixDropdownMenu.Item>
+  );
+}
```
Comment (gazzdingo) [inline packages/front-end/components/Radix/Dropdown.tsx:88]:
```suggestion
}

```

**Review by msamper — APPROVED** [review]
Tested and looks good!

**Pull request merged**